test/Microsoft.TestPlatform.AcceptanceTests/BlameDataCollectorTests.cs

```diff
@@ -64,7 +64,7 @@ public void BlameDataCollectorShouldOutputDumpFile(RunnerInfo runnerInfo)
     {
 
         SetTestEnvironment(_testEnvironment, runnerInfo);
-        var assemblyPaths = BuildMultipleAssemblyPath("SimpleTestProject3.dll").Trim('\"');
+        var assemblyPaths = BuildMultipleAssemblyPath("SimpleTestProject3.dll");
```

Comment:
Yes, but I did not want to change every single file, but many of those methods only exist because all the other methods use strings, and so they need to have a different name. And because ExecuteVSTestConsole takes sources as string, so BuildMultipleAssemblyPath takes one or more sources, and concatenates them into one string. There is a lot to clean up in IntegrationTestBase.